src/data-dictionary/events/K8sContainerSample/aa_template.md

```diff
@@ -0,0 +1,9 @@
+---
+name: 
+type: attribute
+units:
+events:
```

Comment:
missing a name

Reply:
This is a template that has been removed.